lib/chef/provider.rb

```diff
@@ -390,9 +390,6 @@ def action_#{name}
           end
         end
       end
-
-      require "chef/dsl/recipe"
-      include Chef::DSL::Recipe
```

Comment:
Those should be covered by inheriting from LWRPBase:

https://github.com/chef/chef/blob/master/lib/chef/provider/lwrp_base.rb#L32